src/host/screenInfo.hpp

```diff
@@ -169,12 +169,13 @@ class SCREEN_INFORMATION : public ConsoleObjectHeader, public Microsoft::Console
     BYTE WriteConsoleDbcsLeadByte[2];
     BYTE FillOutDbcsLeadChar;
     WCHAR LineChar[6];
-#define UPPER_LEFT_CORNER 0
-#define UPPER_RIGHT_CORNER 1
-#define HORIZONTAL_LINE 2
-#define VERTICAL_LINE 3
-#define BOTTOM_LEFT_CORNER 4
-#define BOTTOM_RIGHT_CORNER 5
+
+    static constexpr auto UPPER_LEFT_CORNER = 0;

```

Comment:
Maybe we should file a follow-up work item to turn this into a proper enum, instead of _whatever this is_.

Reply:
(moved to #3366)

Reply:
The standard I was trying to apply was that macros would be `SCARY_CAPS` and constexprs would be `PascalCase`.

Reply:
@j4james has a PR that just gets rid of this layer of indirection, which seems fine to me. #3371